src/scss/_variables.scss

```diff
@@ -0,0 +1,6 @@
+//TODO: add new colors to palette so we can set usecases (oColorsSetUseCase only accepts o-colors palette colors), pending design input
```

Comment:
We can create new colors in the palette that will only apply to the registry-ui, but we still need to know what design is after